src/dispatch/static/dispatch/src/plugin/store.js

```diff
@@ -74,6 +75,14 @@ const actions = {
       })
   }, 500),
   createEditShow({ commit }, plugin) {
+    if (plugin && plugin.broken) {
+      commit(
+        "notification_backend/addBeNotification",
+        { text: "Plugin not installed correctly.", type: "error" },
```

Comment:
Maybe we could point the user to the logs, since we log the actual error there.

```suggestion
        { text: "Plugin not installed correctly. Please review the Dispatch logs or contact your Dispatch Administrator", type: "error" },
```

Or something like that.